src/components/DateRangePicker.jsx

```diff
@@ -345,6 +345,11 @@ export default class DateRangePicker extends React.Component {
             withFullScreenPortal={withFullScreenPortal}
             onDatesChange={onDatesChange}
             onFocusChange={onFocusChange}
+            focusDayPicker={() => {
```

Comment:
Makes sense to move the focusing into `DayPicker`. I think this common parent will still need to coordinate between the input and daypicker though because they're in isolated subtrees.